src/query-compiler/default-query-compiler.ts

```diff
@@ -907,6 +907,12 @@ export class DefaultQueryCompiler
       this.append(')')
     }
 
+    if (node.include) {
+      this.append(' INCLUDE (')
+      this.compileList(node.include)
+      this.append(') ')
```

Comment:
Remove extra space from the end. Each conditional part adds a space at the beginning and not the end.